src/types.jl

```diff
@@ -71,8 +71,34 @@ function FrameCode(scope, src::CodeInfo; generator=false, optimize=true)
         src = copy_codeinfo(src)
         methodtables = Vector{Union{Compiled,TypeMapEntry}}(undef, length(src.code))
     end
+    breakpoint_pcs = Int[]
+    for (i, pc_expr) in enumerate(src.code)
+        if pc_expr == :($(QuoteNode(getproperty))($JuliaInterpreter, :__BREAKPOINT_MARKER__))
+            push!(breakpoint_pcs, i)
+        end
+    end
     breakpoints = Vector{BreakpointState}(undef, length(src.code))
+    if !isempty(breakpoint_pcs)
+        ssalookup = collect(1:length(src.codelocs))
+        # Renumber SSA values in preparation for deleting breakpoint markers
```

Comment:
Could replace the statement with `nothing` so numbers don't change?

Reply:
Didn't know that was allowed. Is it possible to set a breakpoint on a `nothing` statement?

Reply:
The reason I did the whole `deleteat!` for the statements was so that the debugger wouldn't stop at the `@bp` statement and print that it would run `nothing`. However, I can just work around that on the Debugger side: https://github.com/JuliaDebug/Debugger.jl/pull/61.

Reply:
I'm fine with either approach, whichever you prefer.